test-driver/src/Compare.ts

```diff
@@ -218,11 +218,16 @@ export class Compare {
     // parse embedded stream into events
     private parseEmbeddedStream(event: IonEvent): any {
         let container: any = [];
+        let value = "";
         for (let j = 0; j < event.ionValue.length - 1; j++) {
-            if(event.ionValue[j].eventType == IonEventType.STREAM_END)
+            if(event.ionValue[j].eventType == IonEventType.STREAM_END) {
```

Comment:
I also expected `prettier` to add a space between the `if` and the `(`.

Reply:
I just checked the prettier & tslint isn't added to check for test-driver and test folders. Seems like I need to create another PR for that.

Reply:
That explains it, thanks for checking!